script/src/syscalls/mod.rs

```diff
@@ -425,8 +425,8 @@ mod tests {
 
     proptest! {
         #[test]
-        fn test_load_cell_partial(ref data in any_with::<Vec<u8>>(size_range(1000).lift())) {
-            _test_load_cell_partial(data)?;
+        fn test_load_cell_partial(ref data in any_with::<Vec<u8>>(size_range(1000).lift()), offset in any::<u64>()) {
```

Comment:
We should indicate a range for offset before we can rely on `proptest` default behavior.

Reply:
Updated, please review again, thanks.